config/crd/bases/cluster.x-k8s.io_machinedeployments.yaml

```diff
@@ -18,7 +18,6 @@ spec:
     shortNames:
     - md
     singular: machinedeployment
-  preserveUnknownFields: false
```

Comment:
We likely don't want to drop this, it may require updating the kubebuilder annotation for MachineDeployment to set `crd:preserveUnknownFields=false` as per https://github.com/kubernetes-sigs/controller-tools/pull/362